routes/api/users.js

```diff
@@ -0,0 +1,35 @@
+const router = require("express").Router();
+const oktaClient = require('../../lib/oktaClient');
+
```

Comment:
Is this written in this format vs router.route("/").post ... because it is out of the box?

Reply:
I think a similar format could work as you did with podcast routes. I am honostly a little terrified to change anything from this user registration